requirements-dreamstudio.txt

```diff
@@ -0,0 +1 @@
+stability-sdk
```

Comment:
Either need to include `realesrgan` or disable upscaling panel.

Reply:
Disabled upscaling panel when the DreamStudio backend is selected, as well as other unsupported options.